tmt/steps/provision/__init__.py

```diff
@@ -18,6 +18,8 @@
 CONNECTION_TIMEOUT = 60 * 4
 # Wait time when reboot happens in seconds
 SSH_INITIAL_WAIT_TIME = 5
+# Default rsync options
+DEFAULT_RSYNC_OPTIONS = "-Rrz --links --safe-links --delete"
```

Comment:
If I may:

* I'd either convert this string into a list of options or apply `split()` right here. IIUIC, whenever used, there's `.split()` call anyway. So:

```python
DEFAULT_RSYNC_OPTIONS = "-Rrz --links --safe-links --delete".split()
# or
DEFAULT_RSYNC_OPTIONS = ["-Rrz", "--links", "--safe-links", "--delete"]
```

If you think default options variable is better specified as a string, `.split()` won't harm the readability while making variable ready to be used with any kind of command-ish iterable. No need to remember one has to call `split()` before use.

* I'd separate options, i.e. `"-R -r -z ..."`, to allow easier removal of default options. If, for whatever reason, `tmt` would need to drop `-R`, separate options are safer when running `DEFAULT_RSYNC_OPTIONS.replace('-R', '')` (or, with listified version, something like `DEFAULT_RSYNC_OPTION.remove('-R')`).

Reply:
+1, thanks for the suggestion. I think I'd rather remove the `.split()` and make the options a list literal.